__tests__/app.test.js

```diff
@@ -0,0 +1,27 @@
+const db = require("../db/connection.js");
+const seed = require("../db/seeds/seed.js");
+const request = require("supertest");
+const app = require("../app.js");
+const testData = require("../db/data/test-data/index.js");
+
+beforeEach(() => seed(testData));
+
+afterAll(() => db.end());
+
+describe("/api/categories", () => {
+  test("GET -200: responds with an array of category objects with slug and description properties", () => {
+    return request(app)
+      .get("/api/categories")
+      .expect(200)
+      .then((res) => {
+        expect(res.body.categories.length).toBe(4);
+        res.body.categories.forEach((category) => {
+          expect(typeof category).toBe("object");
```

Comment:
Similar to the previous comment, this assertion is covered by the following assertion by using .toMatchObject